tests/conftest.py

```diff
@@ -33,3 +34,22 @@ def tracking_uri_mock(tmpdir, request):
         mlflow.set_tracking_uri(None)
         if "notrackingurimock" not in request.keywords:
             del os.environ["MLFLOW_TRACKING_URI"]
+
+
+@pytest.fixture(autouse=True, scope="session")
+def enable_test_mode_by_default_for_autologging_integrations():
+    """
+    Run all MLflow tests in autologging test mode, ensuring that errors in autologging patch code
+    are raised and detected. For more information about autologging test mode, see the docstring
+    for :py:func:`mlflow.utils.autologging_utils._is_testing()`.
+    """
+    try:
+        prev_env_var_value = os.environ.pop("MLFLOW_AUTOLOGGING_TESTING", None)
```

Comment:
Should we de-hardcode `_spark_autologging` by putting it in a constant variable?

Reply:
Converted `MLFLOW_AUTOLOGGING_TESTING` to a constant value defined in `autologging_utils`. Good call!

Reply:
Great!